Podfile

```diff
@@ -176,7 +176,8 @@ target 'WordPress' do
 
     pod 'Gridicons', '~> 0.16'
 
-    pod 'WordPressAuthenticator', '~> 1.10.4'
+    pod 'WordPressAuthenticator', '~> 1.10.5-beta.3'
+    # pod 'WordPressAuthenticator', :git => 'git@github.com:wordpress-mobile/WordPressAuthenticator-iOS.git', :branch => 'fix/login-voiceover-enhancements'
```

Comment:
nit: do we usually keep the commented out pod link for development?

Reply:
Yes. 🙂 There's a lot of them in this file.